rcsbapi/data/query.py

```diff
@@ -35,29 +35,29 @@ def __init__(self, input_ids: Union[List[str], Dict[Any, Any]], input_type: str,
                 self.__input_ids_list = input_ids
         self.__response = None
 
-    def get_input_ids(self):
+    def get_input_ids(self) -> Union[List[str], Dict[str, str]]:
```

Comment:
changing to include the case where user puts a list in a dictionary.

ex: {'instance_ids': ['1NDO.A', '4HHB']}